lock-api-objects/src/main/java/com/palantir/lock/watch/LockWatchingRpcClient.java

```diff
@@ -31,10 +31,6 @@
     @Path("start-watching")
     void startWatching(@PathParam("namespace") String namespace, LockWatchRequest lockWatchRequest);
 
-    @POST
```

Comment:
why is this class in `lock-api-objects`? it doesn't appear in an actual object does it..?